common/entity/ParticipationCertificate.ts

```diff
@@ -56,5 +56,6 @@ export class ParticipationCertificate {
     @JoinColumn()
     pupil: Pupil;
 
-
+    @Column({ default: false })
+    ongoingLessons: boolean;
```

Comment:
We will probably need a migration before deployment @geroembser?

Reply:
Yes, we should do that. It would be great if that would be part of this PR as well. But as we've done in the past, I'd also be happy to do the migration for you again if there's possibly all the overhead with the containers. So just let me know, it'll be done quickly with my setup if everything else is ok.

Reply:
@geroembser yes, that would be nice, the device on which I could possibly do that myself is currently 600km away ...